spyder/config/appearance.py

```diff
@@ -0,0 +1,437 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Spyder appearance configuration
+"""
+
+import os
+import sys
+
+from spyder.config.fonts import MEDIUM, MONOSPACE, SANS_SERIF, SMALL
+
+WIN = os.name == 'nt'
+LINUX = sys.platform.startswith('linux')
+
+# Default themes
+APPEARANCE = {
+    'icon_theme': 'spyder 3',
+    # Global Spyder fonts
+    'font/family': MONOSPACE,
+    'font/size': MEDIUM,
+    'font/italic': False,
+    'font/bold': False,
+    'rich_font/family': SANS_SERIF,
+    'rich_font/size': SMALL if (LINUX or WIN) else MEDIUM,
+    'rich_font/italic': False,
+    'rich_font/bold': False,
+    'ui_theme': 'automatic',
+    'names': ['emacs', 'idle', 'monokai', 'pydev', 'scintilla',
+              'spyder', 'spyder/dark', 'zenburn', 'solarized/light',
+              'solarized/dark', 'inkpot', 'minimal', 'nightlion/aptana/theme',
+              'notepad++', 'oblivion', 'obsidian', 'pastel', 'retta',
+              'roboticket', 'sublime/te/mo/ex', 'vibrant/ink'],
```

Comment:
Let's rename `sublime/te/mo/ex` to `sublime-monokai/extended` and  `vibrant/ink` to `vibrant-ink`.